tools/ci-scripts/check-semver-hazards

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# This script patches the new runtime crates into an old AWS SDK and runs tests
+# to check for semver hazards, such as a `Storable` being in an unstable runtime crate.
```

Comment:
should note how to set this up (notably, $PWD/aws-sdk-rust, and smithy-rs need to exist)